.circleci/config.yml

```diff
@@ -184,32 +206,40 @@ jobs:
       - restore_cache:
           keys:
             - mix-cache-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}
-      - run:
-          name: Mix build release
-          command: mix do certs.dev, distillery.release
-      - save_cache:
-          key: certs-cache-{{ .Environment.CACHE_VERSION }}-{{ .Branch }}-{{ .Revision }}
-          paths:
-            - "priv"
-      - run:
-          name : Build tar archive
-          command: tar -czf mongoose_push.tar.gz -C _build/${MIX_ENV}/rel/mongoose_push .
       - run: *DOCKER_TAGS
       - run:
           name: Login to DockerHub
           command: docker login -u "${DOCKERHUB_USER}" -p "${DOCKERHUB_PASS}"
       - run:
           name: Build docker image
-          command: docker build -f Dockerfile.release --build-arg MIX_ENV=prod -t $DOCKERHUB_REPOSITORY/$DOCKER_IMAGE:$DOCKER_TAG .
+          command: docker build -f Dockerfile --build-arg MIX_ENV=prod -t mpush:latest .
+      - run:
+          name: Push docker image but do not tag it as the latest
+          command: docker tag mpush:latest $DOCKERHUB_REPOSITORY/$DOCKER_IMAGE:$DOCKER_TAG
```

Comment:
Would it be possible and make sense to push the docker image only after integration tests pass?

Reply:
Actually the whole point here was to make it possible for developer to run this docker image locally and see why tests fail, without need for local image building.

Reply:
One thing I was thinking of here is to maybe create a separate dockerhub repo for those "development" images.

Reply:
All right, that explains. In the end even if tests fails, usually the issue is fixed in the PR so the broken image will be overwritten.